src/mcp/server/lowlevel/server.py

```diff
@@ -104,6 +105,9 @@ async def main():
 # This will be properly typed in each Server instance's context
 request_ctx: contextvars.ContextVar[RequestContext[ServerSession, Any, Any]] = contextvars.ContextVar("request_ctx")
 
+# Context variable to hold the current ServerSession, accessible by notification handlers
+current_session_ctx: contextvars.ContextVar[ServerSession] = contextvars.ContextVar("current_server_session")
+
```

Comment:
What's the motivation for creating this session variable to pass through to the handlers?

Is it only to be able to call `ServerSession.list_changed()`? The tests don't really demonstrate how it will be used.